lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -130,6 +131,36 @@ def add_pods_library
           end
         end
 
+        # Add a reference to pods' resources (like xcassets) to the user target
+        #
+        # @return [void]
```

Comment:
Since this is called from within `project_is_dirty` above, it should return a true value if it's actually made modifications to a user project so we properly track if there has been new changes.